plugins/pyenv-virtualenvwrapper/pyenv-virtualenvwrapper.plugin.zsh

```diff
@@ -0,0 +1,51 @@
+eval "$(pyenv init -)"
+eval "$(pyenv-sh-virtualenvwrapper -)"
+
+  if [[ ! $DISABLE_VENV_CD -eq 1 ]]; then
```

Comment:
Rather than using `!` to negate and `-eq` to test for equality, use `-ne`:

`if [[ $DISABLE_VENV_CD -ne 1 ]];`

It's more readable that way.